android/src/main/java/com/thebluealliance/androidclient/fragments/tasks/UpdateUserModelSettingsTaskFragment.java

```diff
@@ -17,11 +17,13 @@
     private ModelNotificationFavoriteSettings settings;
 
     public UpdateUserModelSettingsTaskFragment() {
-
+        settings = new ModelNotificationFavoriteSettings();
     }
 
     public UpdateUserModelSettingsTaskFragment(ModelNotificationFavoriteSettings settings) {
         this.settings = settings;
+        // TODO(jerry): Call Fragment#setArguments(Bundle) to stash settings so the 0-arg
+        // constructor can call getArguments() to get them?
```

Comment:
setArguments() requires making ModelNotificationFavoriteSettings Parcelable so it can serialize into a Bundle, right? Will that work? I.e. UpdateUserModelSettings won't try to return values by storing them into that instance?